src/iso19111/factory.cpp

```diff
@@ -45,6 +45,8 @@
 #include "proj/internal/lru_cache.hpp"
 #include "proj/internal/tracing.hpp"
 
+#include "sqlite3.hpp"
```

Comment:
should this be a system include?

Reply:
renamed to sqlite3_utils.hpp to avoid any confusion